src/code/audio_synthesis.c

```diff
@@ -637,6 +657,8 @@ Acmd* AudioSynth_DoOneAudioUpdate(s16* aiBuf, s32 aiBufLen, Acmd* cmd, s32 updat
             if (reverb->filterLeft != NULL || reverb->filterRight != NULL) {
                 cmd = AudioSynth_FilterReverb(cmd, aiBufLen * 2, reverb);
             }
+
+            // Saves the wet channel sample from DMEM (DMEM_WET_LEFT_CH) into (ringBuffer) DRAM for future use
```

Comment:
```suggestion
            // Saves the wet channel sample from DMEM (DMEM_WET_LEFT_CH) into DRAM (ringBuffer) for future use
```
also "RDRAM" is more common (Rambus Dynamic Random Access Memory, apparently)